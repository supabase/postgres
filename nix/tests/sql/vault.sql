select
  1
from
  vault.create_secret('my_s3kre3t');

select
  1
from
  vault.create_secret(
    'another_s3kre3t',
    'unique_name',
    'This is the description'
  );

insert into vault.secrets (secret)
values
  ('s3kre3t_k3y');

select
  name,
  description
from
  vault.decrypted_secrets 
order by
  created_at desc 
limit
  3;
 


