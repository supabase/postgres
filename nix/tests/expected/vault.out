select
  1
from
  vault.create_secret('my_s3kre3t');
 ?column? 
----------
        1
(1 row)

select
  1
from
  vault.create_secret(
    'another_s3kre3t',
    'unique_name',
    'This is the description'
  );
 ?column? 
----------
        1
(1 row)

insert into vault.secrets (secret)
values
  ('s3kre3t_k3y');
select
  name,
  description
from
  vault.decrypted_secrets 
order by
  created_at desc 
limit
  3;
    name     |       description       
-------------+-------------------------
             | 
 unique_name | This is the description
             | 
(3 rows)

 
